extension/scripts/features/bank-investment/ttBankInvestment.js

```diff
@@ -0,0 +1,210 @@
+					class: "bank-investment-money-cell-total",
+					text: formatNumber(bankMoneyData.total, { currency: true, decimals: 0 }),
+				}),
+				document.newElement({
+					type: "div",
+					class: "bank-investment-money-cell-per-day",
+					text: formatNumber(bankMoneyData.daily, { currency: true, decimals: 0 }),
+				}),
+			],
+		});
+
+		return {
+			element,
+			dispose: () => {},
+		};
+	}
+
+	function createBankInvestmentContainer(bankAprInfo) {
+		const tableColumnsDefs = [
+			{
+				id: "period",
+				title: "Period",
+				width: 120,
+				cellRenderer: "string",
+				sortable: true,
+			},
+			{
+				id: "regular",
+				title: "Regular",
+				width: 110,
+				cellRenderer: "bankMoney",
+			},
+			{
+				id: "tcbOnly",
+				title: "TCB Only",
+				width: 110,
+				cellRenderer: "bankMoney",
+			},
+			{
+				id: "meritsOnly",
+				title: "10/10 Merits Only",
+				width: 115,
+				cellRenderer: "bankMoney",
+			},
+			{
+				id: "meritsAndTcb",
+				title: "10/10 Merits + TCB",
+				width: 125,
+				cellRenderer: "bankMoney",
+			},
+		];
+		const tableRowsData = Object.values(PERIOD_TYPE).map((period) => _createRow(period));
+		const bestPeriod = tableRowsData.reduce((maxRow, row) => (row.regular.daily > maxRow.regular.daily ? row : maxRow), tableRowsData[0]).period;
+		const customCellRenderers = {
+			bankMoney: bankMoneyCellRenderer,
+		};
+
+		const table = createTable(tableColumnsDefs, tableRowsData, {
+			cellRenderers: customCellRenderers,
+			rowClass: (rowData) => (rowData.period === bestPeriod ? "tt-bank-investment-selected-row" : ""),
+			stretchColumns: true,
+		});
+
+		const { content, container } = createContainer("Bank Investment - Based on 2b investment", {
+			previousElement: document.find(".content-wrapper > .delimiter-999"),
+			class: "tt-bank-investment-container",
+		});
+		content.appendChild(table.element);
+
+		function dispose() {
+			table.dispose();
+			container.remove();
+		}
+
+		function _createRow(period) {
+			return {
+				period: PERIOD_DESC[period],
+				regular: _getMoneyInfo(period, []),
+				tcbOnly: _getMoneyInfo(period, [INVESTMENTS_BONUSES.TCB]),
+				meritsOnly: _getMoneyInfo(period, [INVESTMENTS_BONUSES.Merit]),
+				meritsAndTcb: _getMoneyInfo(period, [INVESTMENTS_BONUSES.TCB, INVESTMENTS_BONUSES.Merit]),
+			};
+		}
+
+		function _getMoneyInfo(period, bonuses) {
+			const apr = parseFloat(bankAprInfo[period]);
+			const aprPercent = apr / 100;
+			const totalBonusRatio = bonuses.reduce((total, bonus) => total * (1 + BONUSES_RATIO[bonus]), 1);
+			const aprWithBonus = aprPercent * totalBonusRatio;
+			const profitPerDayRatio = (aprWithBonus / DAYS_IN_YEAR) * DAYS[period];
+
+			const total = profitPerDayRatio.toFixed(4) * BALANCE;
+			const daily = (total / DAYS[period]).toFixed();
+
+			return {
+				total,
+				daily,
+			};
+		}
+
+		return {
+			dispose,
+		};
+	}
+
+	function createBankInvestmentFacade() {
+		const investmentTimeLeftElement = document.querySelector("p.m-clear");
```

Comment:
We use `document.find`. Yes, it uses `querySelector` internally, but we should be consistent with our usage.